modules/post/multi/manage/screenshare.rb

```diff
@@ -0,0 +1,170 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info={})
+    super( update_info( info,
```

Comment:
Could we rebase this PR and run `rubocop -a modules/post/multi/manage/screenshare.rb` 🎉